waltz-server/src/main/java/com/wepay/waltz/server/internal/Partition.java

```diff
@@ -407,7 +407,18 @@ public void resumePausedFeedContexts() throws StoreException {
     private boolean isValid(PartitionClient client) {
         synchronized (partitionClientSeqNums) {
             Long currentSeqNum = partitionClientSeqNums.get(client.clientId());
-            return currentSeqNum != null && currentSeqNum.equals(client.seqNum());
+            if (currentSeqNum == null) {
+                logger.info(String.format("CurrentSeqNum is null. Message not valid. ClientId: %s, PartitionId: %d",
```

Comment:
I would rather say "PartitionClient is not valid" instead of "Message not valid".
Lets also print seqNum of the received PartitionClient.